### ISSUE TITLE
`introspect --force` does not work

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#3591]
When playing around with https://github.com/prisma/studio/issues/541 (that schema was in `prisma/schema.prisma`) I observed the following:
```
C:\Users\Jan\Documents\throwaway
λ prisma introspect --url="mysql://root:prisma@localhost:3306/mapos"

Introspecting …

Error: P1012 Introspection failed as your current Prisma schema file is invalid

Please fix your current schema manually, use prisma validate to confirm it is valid and then run this command again.
Or run this command with the --force flag to ignore your current schema and overwrite it. All local modifications will be lost.


C:\Users\Jan\Documents\throwaway
λ prisma introspect --url="mysql://root:prisma@localhost:3306/mapos" --force

Introspecting …

Error: P1012 Introspection failed as your current Prisma schema file is invalid

Please fix your current schema manually, use prisma validate to confirm it is valid and then run this command again.
Or run this command with the --force flag to ignore your current schema and overwrite it. All local modifications will be lost.
```

Expected:
When using `--force` it works and does not tell me to use the same command I am already running.

Workaround:
Delete the schema file manually.

Version:
```
λ prisma -v
@prisma/cli          : 2.7.0-dev.57
Current platform     : windows
Query Engine         : query-engine a6a6ecba74cbaeee0756c312b1b0f2502b27e49c (at ..\..\..\..\ProgramData\nvm\v12.16.0\node_modules\@prisma\cli\query-engine-windows.exe)
Migration Engine     : migration-engine-cli a6a6ecba74cbaeee0756c312b1b0f2502b27e49c (at ..\..\..\..\ProgramData\nvm\v12.16.0\node_modules\@prisma\cli\migration-engine-windows.exe)
Introspection Engine : introspection-core a6a6ecba74cbaeee0756c312b1b0f2502b27e49c (at ..\..\..\..\ProgramData\nvm\v12.16.0\node_modules\@prisma\cli\introspection-engine-windows.exe)
Format Binary        : prisma-fmt a6a6ecba74cbaeee0756c312b1b0f2502b27e49c (at ..\..\..\..\ProgramData\nvm\v12.16.0\node_modules\@prisma\cli\prisma-fmt-windows.exe)
Studio               : 0.281.0

**Comment by janpio** [comment]
Ugh, same behavior when already having the datasource in the schema file unfortunately:

```
C:\Users\Jan\Documents\throwaway
λ prisma introspect

Introspecting based on datasource defined in schema.prisma …

Error: P1012 Introspection failed as your current Prisma schema file is invalid

Please fix your current schema manually, use prisma validate to confirm it is valid and then run this command again.
Or run this command with the --force flag to ignore your current schema and overwrite it. All local modifications will be lost.


C:\Users\Jan\Documents\throwaway
λ prisma introspect --force

Introspecting based on datasource defined in schema.prisma …

Error: P1012 Introspection failed as your current Prisma schema file is invalid

Please fix your current schema manually, use prisma validate to confirm it is valid and then run this command again.
Or run this command with the --force flag to ignore your current schema and overwrite it. All local modifications will be lost.
```

**Comment by pantharshit00** [comment]
I can confirm it doesn't override the warning and writes the file with `--force`

Reproduction:

Just have an invalid schema and try running `prisma introspect --force`

**Comment by Jolg42** [comment]
Fixed in `2.7.0-dev.73` and will be part of 2.7.0 ✅
